test/test_qdmi.cpp

```diff
@@ -221,7 +222,7 @@ measure q -> c;
                                   static_cast<int>(sizeof(int)) * size,
                                   val_vec.data(), nullptr),
             QDMI_SUCCESS);
-  std::map<std::string, int> results;
+  std::unordered_map<std::string, int> results;
```

Comment:
```suggestion
  std::map<std::string, int> results;
```
This change will be obsolete based on the changes in #82.